etstool.py

```diff
@@ -234,8 +235,7 @@ def cleanup(runtime, environment):
     """
     parameters = get_parameters(runtime, environment)
     commands = [
-        "edm run -e {environment} -- python setup.py clean",
```

Comment:
Another unrelated change piggybacking on this one: `python setup.py clean` isn't doing anything useful here, and it's the only command that calls `setup.py` directly instead of going through `pip`.